src/plugins/dam/components/search-assets-screen/index.js

```diff
@@ -96,11 +99,12 @@ function SearchAssetsScreen(props) {
                 const canUpdate = policy.isGranted(`${schema.getQName()}:update`);
                 const seq = node.get('gallery_seq');
                 const transcodingStatus = node.get('transcoding_status');
+                const handleNodeClick = nodeClickNavigator(navigate, node);
```

Comment:
@rsumilang call it `const handleRowClick = createRowClickHandler(navigate, node);`